portal-user-datastore-inmemory/src/main/java/com/telecominfraproject/wlan/portaluser/datastore/inmemory/PortalUserDatastoreInMemory.java

```diff
@@ -156,6 +156,19 @@ public PortalUser getByUsernameOrNull(int customerId, String username) {
     	return ret;
     }
     
+    @Override
+    public List<Integer> getIdsFromUsername(String username) {
```

Comment:
I would call it `getCustomerIdsForUsername`

Reply:
Changed all the methods to getCustomerIdsForUsername for clarity